Wireframe/css/style.css

```diff
@@ -0,0 +1,57 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    margin: 0;
+    padding: 0 0 80px 0;
+}
+header{
+    text-align: center;
+    padding: 20px;
+    background-color: #f4f1f1;;
+}
+
+.wrapper{
+    padding: 20px;
+    width: 80%;
+    margin: 0 auto;
+}
+
+article:first-child {
+    display: block;
+    margin-bottom: 20px;
+    margin: 0 auto 2rem auto;
```

Comment:
This CSS has more importance that the line above. The values `0 auto 2rem auto` are equal to `margin-top: 0`, `margin-right: auto`, `margin-bottom: 2rem` and `margin-left: auto`

So this articles margin-bottom is 2rem because line 22 overrides the line 21 which is `margin-bottom: 20px`

Happy to explain in more detail, but generally css further down your file has more importance than what is written above.